MergerLogic/Clients/HeartbeatClient.cs

```diff
@@ -9,10 +9,10 @@ public class HeartbeatClient : IHeartbeatClient
         private readonly ILogger _logger;
         private readonly IConfigurationManager _configurationManager;
         private readonly IHttpRequestUtils _httpClient;
-        private System.Timers.Timer _timer;
+        private System.Timers.Timer? _timer;
```

Comment:
Because of other changes made, Timer is no longer `nullable`.

Reply:
fixed